spec.md

```diff
@@ -458,7 +457,32 @@ $ curl http://username:password@broker-url/v2/service_instances/:instance_id?acc
 | 409 Conflict | MUST be returned if a service instance with the same id already exists but with different attributes. The expected response body is `{}`, though the description field MAY be used to return a user-facing error message, as described in [Broker Errors](#broker-errors). |
 | 422 Unprocessable Entity | MUST be returned if the broker only supports asynchronous provisioning for the requested plan and the request did not include `?accepts_incomplete=true`. The expected response body is: `{ "error": "AsyncRequired", "description": "This service plan requires client support for asynchronous service operations." }`, as described below. |
 
-Responses with any other status code will be interpreted as a failure. Brokers can include a user-facing message in the `description` field; for details see [Broker Errors](#broker-errors).
+Responses with any other status code will be interpreted as a failure, and in some cases, the broker client SHOULD initiate orphan mitigation. See the next section for details on orphan mitigation. Also, brokers can include a user-facing message in the `description` field; for details see [Broker Errors](#broker-errors).
+
+##### Orphans
+
+The platform marketplace is the source of truth for service instances. Service brokers are expected to have successfully provisioned all the service instances and bindings that the marketplace knows about, and none that it doesn't.
+
+Orphans can result if the broker does not return a response before a provision request from the marketplace times out (typically 60 seconds). For example, if a broker does not return a response to a provision request before the request times out, the broker might eventually succeed in provisioning a service instance after the marketplace considers the request a failure. This results in an orphan service instance on the broker's side.
+
+To mitigate orphan service instances and bindings, the marketplace SHOULD attempt to deprovision resources it cannot be sure were successfully created, and SHOULD keep trying to delete them until the broker responds with a success.
```

Comment:
How does it work with https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#blocking-operations requirement? Do we expect to receive `422 Unprocessable Entity` if the provisioning is still ongoing, and shall retry until we succeed deleting?